### PR TITLE
SOC-5933 : order  Instant messaging in my profile page

### PR DESCRIPTION
**Pull request by seifeddineDaoud**

### CODE REVIEW

File: webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java [hunk 1]
```diff
@@ -220,9 +220,8 @@ public static String getLabel(WebuiRequestContext context, String key) {
   private static void putInfoData(Profile currentProfile, Map<String, Object> infos, String mainKey) {
     List<Map<String, String>> multiValues = getMultiValues(currentProfile, mainKey);
     if (multiValues != null && multiValues.size() > 0) {
-      Map<String, List<String>> mainValue = new HashMap<String, List<String>>();
+      LinkedHashMap<String, List<String>> mainValue = new LinkedHashMap<String, List<String>>();
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: The implementation should not be used on the left side:
Map<String, List<String>> mainValue = new LinkedHashMap<String, List<String>>();

File: webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java [hunk 2]
```diff
@@ -220,9 +220,8 @@ public static String getLabel(WebuiRequestContext context, String key) {
...
       
-      List<String> valuesOfOther = new ArrayList<String>();
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: This method is used for several profile attributes (phones, IMs, URLs). Did you check there is no impacts on all these attributes ?
Also please describe in the PR description why you did this change.

Comment (seifeddineDaoud) [reply webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: This method is also used for the attribute profile (phones), it divides the phones into two categories as for the IMS (category 1: wrokPhone and homePhone, category 2: Other). I think, if we will cancel the division for IMS then we have to cancel it for phones

File: webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java (hunk 1 above)
Comment (seifeddineDaoud) [reply webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: I will change it .

File: webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java (hunk 2 above)
Comment (thomasdelhomenie) [reply webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: I am not sure we want to remove this splitting when displaying these list, that's a functional change. The issue does not consider this case actually. @hghorbel WDYT ?

Comment (hghorbel) [reply webapp/portlet/src/main/java/org/exoplatform/social/user/portlet/UserProfileHelper.java]: We have to unify the display of these attributes, for phone or IM, the order we already have in the edit mode should be the same in the view mode.

**Pull request merged**